nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GenerateTableFetch.java

```diff
@@ -115,20 +128,36 @@ public GenerateTableFetch() {
 
     @OnScheduled
     public void setup(final ProcessContext context) {
+        // The processor is invalid if there is an incoming connection and max-value columns are defined
+        if (context.getProperty(MAX_VALUE_COLUMN_NAMES).isSet() && context.hasIncomingConnection()) {
+            throw new ProcessException("If an incoming connection is supplied, no max-value column names may be specified");
```

Comment:
I got this error while passing incoming flow files to dynamically specify target table name to fetch. But since those target tables has the same column structure, I specified a column name `ID` as `Maximum-value Columns`. What concern is there behind this check condition? I think it'd be useful to specify MAX_VALUE_COLUMN_NAMES by either static string or EL result.